client/me/controller.js

```diff
@@ -303,7 +302,7 @@ export default {
 
 		if ( isWelcome ) {
 			ReactDom.unmountComponentAtNode( document.getElementById( 'secondary' ) );
-			context.layout.setState( { noSidebar: true } );
+			context.store.dispatch( setSection( null, { hasSidebar: false } ) );
```

Comment:
While I agree this can be the case for isLoading, `hasSidebar` is very strongly logically connected to a section it should acompany.
This instance here is one of 3 in entire code where sidebar is set without pointing a section and is - in my opinion - a hack and not entirely clear.
Thats why I didnt set seperate `hasSidebar` action in order to not encourage its usage.

When it comes to `isLoading`, it would be helpful, but code dispatching `isLoading` param is generated dynamically on the server in `sections.js` and I dispatched actions there explicitly through `{ type: SET_SECTION, isLoading: true}`, without actionCreator.
Because that code is created on the server, I didnt want to inlude additional dependencies, because it would be hard to figure out where the action creator came from while reading the soure.